client/components/Forms/EventForm.js

```diff
@@ -229,6 +263,7 @@ EventForm.propTypes = {
   createEvent: PropTypes.func.isRequired,
   formId: PropTypes.string.isRequired,
   dismiss: PropTypes.func.isRequired,
+  categories: PropTypes.arrayOf(PropTypes.object).isRequired,
```

Comment:
Does the category object have a fixed shape? If yes, then use `PropTypes.shape` to define it's shape.

Reply:
No fixed shape

Reply:
Gotcha @chibujax